nvtabular/tf_dataloader.py

```diff
@@ -124,12 +125,13 @@ class KerasSequenceDataset(tf.keras.utils.Sequence):
 
   Parameters
   -------------
-  - file_pattern: str or list(str)
+  - paths_or_dataset: str or list(str)
```

Comment:
Note that the changes in this file are not necessary for this PR, but I left them in (for now) - If @alecgunny will be changing everything here soon anyway, I will just reset this file.